KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -20,8 +20,6 @@
 /* User errors, K1xxx */
 "K1001" = "Could not register user";
 
-"K1002" = "Could not register user (user already exists)";
```

Comment:
I see there are other user registration related errors still here. Are we sure this error is obsolete? We don't register users for the token refresh flow, so the chances of encountering these errors is (or should be) close to none. Just curious why this was removed but not the others.

Reply:
There was no usage of them anywhere in the sdk and most of the were not used in android also and deleted from there.